src/systems/audio-system.js

```diff
@@ -62,7 +62,7 @@ export class AudioSystem {
     if (sceneEl.camera) {
       sceneEl.camera.add(sceneEl.audioListener);
     }
-    sceneEl.addEventListener("camera-set- active", evt => {
+    sceneEl.addEventListener("camera-set-active", evt => {
```

Comment:
Maybe worth logging to see when the body runs. Is this related to not being able to hear things?